.github/workflows/nightly.yml

```diff
@@ -9,6 +9,8 @@ on:
     #        │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
     #        │ │ │ │ │
     - cron: "0 0 * * 0,3"  # Every Sunday and Wednesday at midnight
+  pull_request:
+    branches: [master]
```

Comment:
Should probably revert this and the removals of the `if` clauses before merge, hence this PR has been changed to draft mode